config/environments/production.rb

```diff
@@ -86,16 +86,13 @@
 
   config.action_mailer.default_url_options = { host: "community.recurse.com", protocol: "https" }
 
-  config.action_mailer.eventmachine_smtp_settings = {
+  config.action_mailer.smtp_settings = {
     address: ENV["SMTP_SERVER"],
     port: ENV["SMTP_PORT"],
     user_name: ENV["SMTP_USERNAME"],
     password: ENV["SMTP_PASSWORD"],
+    tls: ENV["SMTP_TLS"].present?,
     authentication: :plain,
```

Comment:
Given that I was going to test this in development anyway, I decided to test migrating from SMTP+STARTTLS to SMTPS.

Once the deploy is done, we can edit the config and set the following:

```
SMTP_PORT=465
SMTP_TLS=true
```

Once that's done, we can deploy a new PR that just sets `tls: true`